slackclient/client.py

```diff
@@ -88,27 +91,29 @@ def api_call(self, method, timeout=None, **kwargs):
         response_body = self.server.api_call(method, timeout=timeout, **kwargs)
         try:
             result = json.loads(response_body)
-        except ValueError as json_decode_error:
```

Comment:
Keep this up here so it's more clear what `Error`s you handle for which lines

Reply:
It was still there, just moved way down into the wrong scope...